daemon/src/collab_settlement_maker.rs

```diff
@@ -195,6 +243,20 @@ impl Actor {
             })
             .await?;
 
+        let timeout = {
+            let this = ctx.address().expect("self to be alive");
+            async move {
+                tokio::time::sleep(INITIATE_TIMEOUT).await;
+
+                this.send(InitiateTimeoutReached {
+                    timeout: INITIATE_TIMEOUT,
+                })
+                .await
+                .expect("can send to ourselves");
```

Comment:
It is unlikely that this happens because our `Tasks` are tied to the lifetime of the actor but I think it would be safer here to just `_` the result. If we can't send this message than the actor is disappeared which means it doesn't matter anymore.

Reply:
Note: I aligned removing `expect` from the other actor impls, see latest commit.